src/Core/Geographic/Extent.js

```diff
@@ -336,4 +336,28 @@ Extent.prototype.merge = function merge(extent) {
     }
 };
 
+/**
+ * mergeCoordinate performs the minimum extension for
+ * a coordinates to belong to the extended
+ *
+ * @param {Coordinates} coordinates  The coordinates to belong
+ */
+Extent.prototype.mergeCoordinate = function mergeCoordinate(coordinates) {
```

Comment:
I'd suggest using the same verb than three.js for easier understanding. [Box3](https://threejs.org/docs/#api/math/Box3) use `expandByPoint` so I propose `expandByCoordinate`.

Reply:
ok 
So I change `merge` by `expandByExtent`?

Reply:
See `Box3`: they have a `union` method to merge 2 box so if yes, you could rename `merge` -> `union`

Reply:
done